partcad/src/partcad/shape.py

```
@@ -19,9 +16,11 @@
 import tempfile
 import threading
 
+from .cache_hash import CacheHash
 from .render import *
```

Ruff (0.8.2)

`from .render import *` used; unable to detect undefined names

(F403)